rust-toolchain

```diff
@@ -1 +1 @@
-stable
```

Comment:
What change requires `nightly`?

Reply:
Using hpos-config-core

Reply:
It was an `ed25519-dalek` dependency. I forget which one.